civis/io/_tables.py

```diff
@@ -727,3 +765,48 @@ def f(x):
 
         fut.add_done_callback(f)
     return fut
+
+
+def _redshift_to_py(civ_client, tableid):
```

Comment:
This library has the convention of naming `APIClient` objects as `client`. You should use that instead of `civ_client` here.